### PR TITLE
Added snapshot testing for UserRequestToAlexaForm component.

### PR DESCRIPTION
**Pull request by s-maheshbabu**
Added snapshot testing for UserRequestToAlexaForm component. Quite straightforward test based on standard snapshot test template.

### CODE REVIEW

File: src/UserRequestToAlexaForm.test.js [hunk 1]
```diff
@@ -11,6 +11,17 @@ it("renders UserRequestToAlexaForm without crashing", () => {
   mount(<UserRequestToAlexaForm />);
 });
 
+it("that it renders correctly without any props (snapshot testing)", () => {
```
Comment (gootla-manasa) [inline src/UserRequestToAlexaForm.test.js]: test description doesnt follow convention. "renders correctly without any props"?

File: src/UserRequestToAlexaForm.test.js [hunk 2]
```diff
@@ -11,6 +11,17 @@ it("renders UserRequestToAlexaForm without crashing", () => {
...
+  const wrapper = mount(<UserRequestToAlexaForm />);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it("that it renders correctly with props (snapshot testing)", () => {
```
Comment (gootla-manasa) [inline src/UserRequestToAlexaForm.test.js]: renders correctly with props?

File: src/UserRequestToAlexaForm.test.js [hunk 3]
```diff
@@ -11,6 +11,17 @@ it("renders UserRequestToAlexaForm without crashing", () => {
...
+  const dummyUserRequest = "A dummy user request";
```
Comment (gootla-manasa) [inline src/UserRequestToAlexaForm.test.js]: This variable is not needed.

File: src/UserRequestToAlexaForm.test.js (hunk 1 above)
Comment (s-maheshbabu) [reply src/UserRequestToAlexaForm.test.js]: Done.

File: src/UserRequestToAlexaForm.test.js (hunk 2 above)
Comment (s-maheshbabu) [reply src/UserRequestToAlexaForm.test.js]: Done.

File: src/UserRequestToAlexaForm.test.js (hunk 3 above)
Comment (s-maheshbabu) [reply src/UserRequestToAlexaForm.test.js]: Moved it inline.

**Pull request merged**